server_environment.yml

```diff
@@ -2,24 +2,25 @@ name: base
 channels:
 - conda-forge/label/vtk_dev
 - conda-forge
-- defaults
 dependencies:
 - python>=3.7
 - pip
 - mesalib
 - ffmpeg
 - vtk
 - traits
+- scipy
+- numpy
+- matplotlib-base
+- pyvista
+- nilearn
+- nibabel
+- nbformat
+- nbclient
+- mffpy>=0.5.7
 - pip:
   - mne
```

Comment:
mne and jupyter are stuck in `pip` because they require / pull in PyQt5, which often breaks headless servers (anything that tries to import PyQt5 will die because Qt5 throws an error)